sql-cli/tests/test___main__.py

```diff
@@ -53,27 +53,15 @@ def test_validate():
     assert result.exit_code == 0
 
 
-def test_run_dag(root_directory_dags):
-    result = runner.invoke(
-        app,
-        ["run", "example_dataframe", root_directory_dags.as_posix(), "--no-is-sql"],
-    )
-    assert result.exit_code == 0
-    result_stdout = get_stdout(result)
-    assert "The worst month was 2020-05" in result_stdout
```

Comment:
@dimberman The goal for sql-cli was to run on a directory of sql files, hence we don't pass a dag_id. We instead generate the dag first and then get the dag id internally from the generated file which is why I removed it this test. We have another test below for testing the run statement.

@tatiana is this correct?

Reply:
@feluelle testing this mode helps ensure that nothing breaks in the run dag regarding DAG running. Only having integration tests for the generate + run flow will make it more difficult to debug in the future.